ckanext/recombinant/controller.py

```diff
@@ -294,3 +298,5 @@ def _process_upload_file(lc, dataset, upload_file, geno):
             raise BadExcelData(
                 _(u"Error while importing data: {0}").format(
                     pgerror))
+    if total_records==0:
+        raise BadExcelData(_("The template updated is empty"))
```

Comment:
`if not total_records:` is more common in python than `==0`

Can we change this text to `The template uploaded is empty`? `updated` sounds pretty awkward to me